CMakeLists.txt

```diff
@@ -25,18 +29,32 @@ function(join result_var)
 endfunction()
 
 function(enable_module target)
+  message(STATUS "FMT_MODULE: Build the CXX-MODULE")
   if (MSVC)
     if (${CMAKE_GENERATOR} STREQUAL "Ninja")
-      set(BMI ${CMAKE_BINARY_DIR}/fmt.ifc)
-    else()
-      set(BMI ${CMAKE_CURRENT_BINARY_DIR}/fmt.ifc)
-    endif()
+      set(_BMI ${CMAKE_BINARY_DIR}/fmt.ifc)
+    else ()
+      set(_BMI ${CMAKE_CURRENT_BINARY_DIR}/fmt.ifc)
+    endif ()
     target_compile_options(${target}
       PRIVATE /interface /dxifcInlineFunctions-
-      INTERFACE /reference fmt=${BMI})
+      INTERFACE /reference fmt=${_BMI})
+  elseif (CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+    set(_BMI ${CMAKE_BINARY_DIR}/fmt.cc.pcm)
+    target_compile_options(${target}
+      PRIVATE -fmodule-output=${_BMI} -x c++-module
+      INTERFACE -fmodule-file=fmt=${_BMI})
+    if (${target} MATCHES "fmt")
+      add_custom_command(
+        TARGET fmt POST_BUILD
+        COMMAND ${CMAKE_COMMAND} -E create_symlink ${PROJECT_BINARY_DIR}/CMakeFiles/fmt.dir/src/fmt.cc.pcm ${_BMI}
+        DEPENDS ${FMT_SOURCES})
+    endif ()
```

Comment:
Only a workaround, not clear why the a 2 PCM files created with same content?
See `TODO.txt` for my build steps.

Reply:
OK, this is fixed now ;-)